feature-libs/pickup-in-store/components/presentational/store/store.component.ts

```diff
@@ -25,4 +25,11 @@ export class StoreComponent {
   toggleOpenHours(): void {
     this.openHoursOpen = !this.openHoursOpen;
   }
+
+  isInStock(): boolean {
```

Comment:
This should be replaced with a variable.

Reply:
Done